docs/src/api/class-browsercontext.md

```diff
@@ -227,7 +227,7 @@ Script to be evaluated in all pages in the browser context.
 Script to be evaluated in all pages in the browser context.
 
 ### param: BrowserContext.addInitScript.arg
-* langs: js
```

Comment:
We dropped the argument in python and java, it only exists in javascript. Let's ditch it in C# too? It is always possible to workaround like this: `"(" + script + ")(" + toJson(arg) + ")"`